pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -161,7 +163,16 @@ func (ka *kubeactions) List(ctx context.Context, oc *api.OpenShiftCluster, group
 		return nil, err
 	}
 
-	return ul.MarshalJSON()
+	if jpath == nil {
+		return ul.MarshalJSON()
+	}
+
+	results, err := jpath.Search(ul)
```

Comment:
please add this functionality to get k8s object, list openshift clusters and list cluster resources for completeness